WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt

```diff
@@ -608,6 +608,31 @@ d_exilarch = {
 	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
 }
 
+d_exilarch_karaite = {
+	color = { 155 216 230 }
+	color2 = { 220 220 0 }
+	
+	capital = 693 # Baghdad
+
+	title = "EXILARCH"
+	foa = "KOHEN_GADOL_FOA"
+	short_name = yes
+	
+	# Parent Religion 
+	religion = karaite
+	
+	culture = mizrahi
+	
+	# mercenary = yes
+
+	# Always exists
+	landless = yes
+	
+	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
+	
+	# mercenary_type = exilarch_karaite_composition
+}
+
```

Comment:
Where did you find this title?

Reply:
In landed_titles.txt in VIET files

Reply:
The latest public release?

Reply:
No, the one I mentioned some time ago

Reply:
I'm not sure on the title, I will have to look into it.

Reply:
Has it been removed from VIET?

Reply:
I think it was removed in the latest version, but that one didn't include all modules either.

Reply:
I think it should be in:
- karaite religion exists in game and this is their religious head
- this title is referenced in many VIET events

Reply:
I think it should, but if you look at the history you added this title was created in the 8th century.